### DISCUSSION TITLE
Show post excerpt / summary on card hover

### DISCUSSION THREAD

**Discussion by roroland** [dailydotdev/daily#508] [Feature Requests]
Hi, would like to see when I hover the card a short summary / excerpt of the post without the need to clicking on it and go to the post to actually understand what is about.
I see many cards that doesn't provide you enough context of what you about to read like this:
<img width="433" alt="Screen Shot 2022-03-24 at 09 30 48" src="https://user-images.githubusercontent.com/200220/159916641-1eacdad6-9d1f-49b4-9ebb-1f267fb1fecf.png">

Is clear what is the subject about, but not if it's an article for beginners or is an interesting topic with more advance knowledge. If I can read some part of it's content here I can optimize my flow without going to articles that aren't that interesting for me.

Thanks

**Comment by nimrodkra** [comment]
@roroland We plan to release two new features that should make the TLDR easily accessible. I won't be based on the hover function (as it's not mobile-friendly), but it should make it much easier to know what the article is about before deciding to read it. You touched a very valid point there, thank you! I'll keep you posted here once those changes are out 🙌
